picard/__init__.py

```diff
@@ -24,12 +24,26 @@
 PICARD_APP_NAME = "Picard"
 PICARD_VERSION = (2, 0, 4, 'dev', 1)
 
-
 # optional build version
 # it should be in the form '<platform>_<YYMMDDHHMMSS>'
 # ie. win32_20140415091256
 PICARD_BUILD_VERSION_STR = ""
 
+PROGRAM_UPDATE_LEVELS = {
+    'stable': {
+        'level': 0,
+        'title': 'Stable releases only',
```

Comment:
You need to mark strings for translation using `N_()` here

Reply:
Will do.  I need to do some reading so I understand the translation stuff better.

Reply:
Ask me if you need, i18n is a rather complex matter.

Reply:
Thanks.  I'll try to do the research on my own first, so as not to waste your time.  That way when I do have a question it should be a short clarification only, and not require you to provide a long explanation.  You have much more important things to do with your time than spending it tutoring me.